tensorflow/examples/udacity/README.md

```diff
@@ -54,6 +54,9 @@ to get the ip of the new virtual machine. To switch from default virtual machine
     
 Note that `docker-machine env tensorflow` outputs some environment variables such like `DOCKER_HOST`. Then your docker client is now connected to the docker host in virtual machine `tensorflow`
 
+* **I'm getting the error - docker: Cannot connect to the Docker daemon. Is the docker daemon running on this host? - when I run 'docker run'.**
+
+This is a permissions issue, and a popular answer is provided for Linux and Max OSX [here](http://stackoverflow.com/questions/21871479/docker-cant-connect-to-docker-daemon) on StackOverflow. Googling the error message does not turn up this result, so it is included here.
```

Comment:
Good idea. I created [a new PR](https://github.com/tensorflow/tensorflow/pull/6408). This can be deleted. Sorry.